### PR TITLE
killall: fix typo

### PR DESCRIPTION
**Pull request by Sunt-ing**
As title. 
Thanks for your efforts in maintaining this project.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/killall.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 - List available signal names (to be used without the 'SIG' prefix):
 
-`killall --list`
+`killall -list`
```
Comment (kbdharun) [inline pages/common/killall.md:13]:
```suggestion
`killall --list`
```
Welcome to tldr, thanks for your contribution. This isn't a typo here long arguments require 2 `-`. We prefer longer arguments in tldr. 1 `-` will be applicable here for `-l` which is a shorter flag.

Comment (Sunt-ing) [reply pages/common/killall.md:13]: Thanks for your welcome. In my MBP M1 laptop, `killall --list` is not applicable, while `killall -list` is.

Comment (Sunt-ing) [reply pages/common/killall.md:13]: image

Comment (kbdharun) [reply pages/common/killall.md:13]: > Thanks for your welcome. In my MBP M1 laptop, `killall --list` is not applicable, while `killall -list` is.

Oh, I will check my laptop and get back to you.

Comment (kbdharun) [reply pages/common/killall.md:13]: Update: Both flags seem to work for me. 

Also, their man page has double arguments so not sure.

image

Comment (Sunt-ing) [reply pages/common/killall.md:13]: Oh, in my laptop, the man page is:
```
KILLALL(1)                                                               General Commands Manual                                                              KILLALL(1)

NAME
     killall – kill processes by name

SYNOPSIS
     killall [-delmsvqz] [-help] [-I] [-u user] [-t tty] [-c procname] [-SIGNAL] [procname ...]
...
```
while Linux man page for killall is:
```
Name

killall - kill processes by name
Synopsis

killall [-Z,--context pattern] [-e,--exact] [-g,--process-group] [-i,--interactive] [-q,--quiet] [-r,--regexp] [-s,--signal signal] [-u,--user user] [-v,--verbose] [-w,--wait] [-I,--ignore-case] [-V,--version] [--] name ...
killall -l
killall -V,--version 
...
```

The implementations are different.
In this case, I have no strong opinion on any decision of this PR.

**Review by pixelcmtd — COMMENTED** [review]
I don't _think_ we have formalized this anywhere, but wherever OSes (usually GNU and BSD) differ, we just leave the example out of the `common` page and make platform-specific pages. I'll just make a new PR doing just that.

**Comment by EmilySeville7cfg** [comment]
@pixelcmtd, please refer to #9527 PR for more details.

**Comment by pixelcmtd** [comment]
This PR was superseded by #9550.

**Pull request closed without merging**